docs/architecture/cloud-native/elastic-search-in-azure.md

```diff
@@ -38,8 +38,6 @@ This chapter presented a detailed look at data in cloud-native systems. We start
 
 - [Event Sourcing pattern](https://docs.microsoft.com/azure/architecture/patterns/event-sourcing)
 
-- [RDBMSs vs. NoSQL Databases: Overview](https://maxivak.com/rdbms-vs-nosql-databases/)
```

Comment:
this link works. are you sure you want to remove it?

Reply:
Maira... Same story. Can access the link directly, but Pandoc, the PDF generator throws a 404 on that link. Removed it so that we could generate the PDF. Please advise

Reply:
It's up to you. If you want this to be removed, then you can resolve the conversation.